inc/plugins/myalerts.php

```diff
@@ -7,7 +7,7 @@
  * @package MyAlerts
  * @author  Euan T. <euan@euantor.com>
  * @license http://opensource.org/licenses/mit-license.php MIT license
- * @version 2.0.3
+ * @version 2.0.4
```

Comment:
This'll end up getting bumped to 2.0.5, but I might just remove it as it's too say to forget 🤣

Reply:
Removed :)